recipes/tb-profiler/meta.yaml

```diff
@@ -37,7 +37,7 @@ requirements:
     - bedtools
     - freebayes
     - samclip
-    - snpeff=4.3
+    - snpeff=5.0
```

Comment:
are you sure this version is working?

Reply:
By looking at the tests it installed **snpeff-5.0-hdfd78af_1**.
```
22:20:35 BIOCONDA INFO (ERR) [Nov 10 22:20:35] SOUT snpeff             bioconda/noarch::snpeff-5.0-hdfd78af_1
```
This version seems to work ok locally for me. Was there an issue with this release of snpEff?

Reply:
Tested on macos where `SnpEff version SnpEff 5.0e` appeared to work